### PR TITLE
MOTECH-2420 Complete program enrollment and update its state

### PR DESCRIPTION
**Pull request by martokarski**
I created action to update program enrollment state. This action should also complete the program enrollment. There are two ways to do this. First is to change state to the terminal state (it automatically trigger program enrollment). Second is to set complete date. I created second action to fully covered requirements.

### CODE REVIEW

**Comment by martokarski** [comment]
test this please

**Comment by martokarski** [comment]
test this please

**Comment by kmadej** [comment]
After conversation with Craig we established that we need only one task action. Please correct this PR

File: openmrs-19/src/main/java/org/motechproject/openmrs19/tasks/impl/OpenMRSActionProxyServiceImpl.java [hunk 1]
```diff
@@ -146,6 +147,27 @@ public void createProgramEnrollment(String configName, String patientUuid, Strin
...
     }
 
+    @Override
+    public void changeStateOfProgramEnrollment(String configName, String programEnrollmentUuid, DateTime programCompletedDate,
+                                               String stateUuid, DateTime startDate) {
+        ProgramEnrollment programEnrollment = new ProgramEnrollment();
+        programEnrollment.setUuid(programEnrollmentUuid);
+        programEnrollment.setDateCompleted(Objects.nonNull(programCompletedDate) ? programCompletedDate.toDate() : null);
+
+        if (!Strings.isNullOrEmpty(stateUuid)) {
```
Comment (kmadej) [inline openmrs-19/src/main/java/org/motechproject/openmrs19/tasks/impl/OpenMRSActionProxyServiceImpl.java]: we are usually using StringUtils from apache.commons, this class is already imported here so I think you should use this

**Comment by kmadej** [comment]
test this please

**Pull request merged**